STEP_Project/OUListContent.js

```diff
@@ -4,6 +4,80 @@ if (chrome.storage !== undefined)
     storageObj = chrome.storage.sync;
 }
 
+/**
+ * This function adds the selected entity to Chrome's storage
+ * @param {HTML tr object} row Each row in the table of entities
+ * @param {string} dataid The data-row-id for each entity
+ */
+function addEntity(row, dataid)
```

Comment:
Won't it be easier to detect which type of entity it is based on URL ?

Reply:
- Added new test cases for url and groups
- Used the link tag in the head to detect URL and distinguish between different entities.